src/gui/properties/speedwidget.cpp

```diff
@@ -44,6 +41,17 @@
 #include "core/preferences.h"
 #include "core/utils/misc.h"
 
+ComboBoxMenuButton::ComboBoxMenuButton(QWidget *parent, QMenu *menu) : QComboBox(parent), m_menu(menu)
```

Comment:
Follow the [style rules](https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md#5-initialization-lists).